package.json

```diff
@@ -60,12 +60,13 @@
     "@types/node": "^12.12.24",
     "@types/react": "^16.9.17",
     "@types/react-paginate": "^6.2.1",
-    "autoprefixer": "^9.7.4",
+    "autoprefixer": "^10.2.5",
```

Comment:
are you forced to update this dependency in the same run? if it causes any troubles, feel free to do that separately